RELEASING.md

```diff
@@ -31,24 +31,18 @@ Update the version's release date and status from the
 [Manage Versions](https://jira.mongodb.org/plugins/servlet/project-config/PHPORM/versions)
 page.
 
-## Update version info
+## Choose a version
```

Comment:
[Trigger the release workflow](https://github.com/mongodb/mongo-php-driver/blob/master/RELEASING.md#trigger-the-release-workflow) from PHPC may be a model for this. It looks like the [PHPLIB instructions](https://github.com/mongodb/mongo-php-library/blob/master/RELEASING.md) also need to be updated to refer to the GitHub workflow.

Reply:
Thanks, I have copied and modified the paragraph from PHPC.